releasenotes/notes/lib-injection-0041b50a263e1c19.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    lib-injection: Use package versions published to PyPI to install the
```

Comment:
do we need this? 🤔 

since this is a workflow/build change.... but it does impact the library images which are version specific...

Reply:
I think we do need this, since technically this is a user-facing change by speeding up installs